nekoyume/Assets/_Scripts/UI/Tween/AnchoredPositionSingleTweener.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using DG.Tweening;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Nekoyume.UI.Tween
+{
+    // todo: `AnchoredPositionSingleTweener`로 바꾸고, 좌표계를 선택할 수 있도록. `RotateSingleTweener` 참고.
+    [RequireComponent(typeof(RectTransform))]
+    public class AnchoredPositionSingleTweener : DOTweenBase
+    {
+        public enum AxisType
```

Comment:
`SerializableAttribute` not necessary?
```suggestion
        [Serializable]
        public enum AxisType
```

Reply:
yes. that is unnecessary.